frontend/src/pages/codebase/Edit.vue

```diff
@@ -10,6 +10,11 @@
                     :required="config.is_replication"
                     help="Is this model a replication of a prior computational model?">
         </c-checkbox>
+        <c-input v-if="is_replication" v-model="replication_text" :errorMsgs="errors.replication_text"
+                 name="replication_url" label="URL / DOI / Citation"
+                 help="URL / DOI / citation for the original model being replicated"
+                :required="config.replication_text">
```

Comment:
Should this be `:required="config.is_replication"` instead so the text is required only if `is_replication` is checked? 

Also, the `:required` for the `is_replication` checkbox above this component seems odd to me, it's only required if it is checked?